docs/ensembl-help/using-ensembl/ensembl-apps/blast/blast.md

```diff
@@ -0,0 +1,66 @@
+---
+slug: blast
+title: How to run BLAST
+description: How to run BLAST queries
+related_articles:
+tags:
+  - blast
+status: draft
+---
+
+# How to run BLAST
+
+BLAST is a sequence similarity search tool that can be used for both DNA and proteins. 
+
+## Selecting BLAST parameters
+
+The target databases available for similarity searches are DNA, transcripts and proteins. Click on the ‘Database’ dropdown menu and choose from these options. 
+
+The following BLAST programs are available:
+BLASTN: nucleotide sequences against nucleotide databases 
+TBLASTX: translated nucleotide sequences against a translated nucleotide database
```

Comment:
You probably wanted this to be a list. As written, this will be transformed by Markdown into a single paragraph of text, like so:

<img width="1042" alt="image" src="https://user-images.githubusercontent.com/6834224/207638852-fe18ef00-cf3a-4dff-9b54-3310cabb9579.png">

Reply:
Thanks for spotting @azangru . Have changed it into a list. Thanks

Reply:
Also, as regards to the text itself — BLASTN and TBLASTX are not the only two available BLAST programs available to the user. The list of  BLAST programs changes depending on the combination of the sequence type of the user's query and the sequence type of the database that the user has chosen for the search. `dna/dna` gives you one option, `dna/protein`, another option, `protein/dna`, yet another option; and `protein/protein` another one still.